doc/riverctl.1.scd

```diff
@@ -268,6 +268,69 @@ A complete list may be found in _/usr/include/linux/input-event-codes.h_
 	and is made available through the _XCURSOR_THEME_ and _XCURSOR_SIZE_
 	environment variables.
 
+## INPUT CONFIGURATION
+
+The _input_ command can be used to create a configuration rule for an input
+device identified by its _name_.
```

Comment:
How does one find out the device names? I'm assuming there's some command to list them but the average user probably isn't aware of it.

Reply:
There isn't a good way currently. The best way is either to read the logs for "new input device" messages, or to hop into sway and use `swaymsg -t get_inputs`.

Reply:
looks like libinput ships a command line tool that can provide this information. `libinput list-devices` prints off a list for me, although the names aren't in the format we expect.

Reply:
I decided to use the same scheme to build the identifiers as Sway does (`<vendor id>:<product id>:<name>`) for the sake of consistency.

The identifier sadly can't be just the advertised device name, as that would lead to even more duplicate identifiers.

`libinput list-devices` does not show the vendor or device id unfortunately.

Reply:
Would it be hard to implement a command like `riverctl devices-info` that return the devices identifier, maybe using json? Could be a better user friendly way than looking at the log